### PR TITLE
OSDOCS-5673: Greenboot release note

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13 only

Issue:
https://issues.redhat.com/browse/OSDOCS-5673

Link to docs preview:
https://58101--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-13-release-notes.html#microshift-4-13-installation

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@ggiguash @dhensel-rh  PTAL at this release note.

**Comment by dhellmann** [comment]
LGTM

**Comment by ggiguash** [comment]
LGTM

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 1]
```diff
@@ -31,14 +31,14 @@ This release adds improvements related to the following components and concepts.
...
 
-//[id="microshift-4-13-new-feat-based-on-RHEL-Edge"]
+//[id="microshift-4-13-new-feat-based-on-{op-system-ostree}"]
 //==== Placeholder for new feat bases on RHEL Edge
 
-//[id="microshift-4-13-installation"]
-//=== Installation (can also include Upgrade when that is available)
+[id="microshift-4-13-installation"]
+=== Installation
+This release includes integration with the Greenboot health check framework. Greenboot is used to assess system health and automate a rollback to the last healthy state in the event of software trouble. You can add the optional Greenboot RPM to your installation. For more information, read the xref:../microshift_running_apps/microshift-greenboot.adoc#microshirt-greenboot[GreenBoot documentation]
```
Comment (lahinson) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:41]:
```suggestion
This release includes integration with the greenboot health check framework. Greenboot assesses system health and automates a rollback to the last healthy state in the event of software trouble. You can add the optional greenboot RPM to your installation. For more information, read the xref:../microshift_running_apps/microshift-greenboot.adoc#microshirt-greenboot[greenboot documentation].
```

- The documentation that you point to uses "greenboot" in all lowercase except where it starts a sentence, so I followed suit in my edit. 
- Not sure if we should spell out "RPM". I looked it up on the [Red Hat abbreviations page](https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon/red_hat_abbreviations_acronyms_and_initialisms_dictionary_wiki/rh_dictionary_letter_r), and apparently it stands for "RPM Package Manager", which a little odd IMHO.

**Review by lahinson — COMMENTED** [review]
Made a couple of minor comments. LGTM overall, though!

**Pull request merged**